doc/source/plans.rst

```diff
@@ -109,6 +109,37 @@ Examples:
     RE.subscribe(bec)
     RE(count([noisy_det], num=5))
 
+.. note::
+
+   Why doesn't :func:`count` have an ``exposure_time`` parameter?
+
+   Modern CCD detectors typically parametrize exposure time with *multiple*
+   parameters. There is no one "exposure time" that can be applied to all
+   detectors.
+
```

Comment:
Perfect place to expand on *why* a common method is a challenge.  Add this paragraph:

```
Scalers also parametrize exposure time with *multiple*
parameters.  The most common term (used by 
`EpicsScaler` and `ScalerCH`) is `det.preset_time`.
```

Do multi-channel analyzers and multi-channel scalers set the time differently.  This is the place to mention *briefly*.